modules/add_new_state/assets/add_new_state.js

```diff
@@ -206,7 +208,13 @@ jQuery(document).ready(function($) {
 					//SPCO.console_log( 'select_id', select_id, false );
 					var set_selected = $(this).attr('id') === state_select_id;
 					// set target select's value to this input's value
-					EE_ANS.add_option_to_dropdown( select_id, new_state.id, new_state.name, set_selected, new_state.country_name );
+					var new_state_option_value = new_state_option_value = new_state.id;;
```

Comment:
copy pasta???

the var name and semicolons are duplicated

Reply:
👽 that's weird. Ugh thanks!